Fixed, and Added small improvements to KS13

HourlyLimit opened the pull request:
## About The Pull Request

I made it so bots now work on station, there is now an axe to remove unwanted windows, I made it so you don't get stuck in the hole in engineering, and added a supermatter from a off course pod for faster station rebuilding so the round is not spent only setting up power.

## Why It's Good For The Game
With the bots when you're a drone you no longer need to clean as much, (but you still need to do a bit that the bots cannot get too), For the axe it's hard to remove the windows in KS13 with how there multi-layered I'm sure you don't want to spend your time dismantling windows. The hole not being able to trap you I'd say is nice as when you randomly lag or walk the wrong way you're not stuck forever. The supermatter would benefit the speed of how fast KS13 can be repaired right now repairing that station will take way way longer than a round. 

## Changelog
![image](https://github.com/tgstation/tgstation/assets/44913068/e6e08e91-c217-4eb8-bddf-39ebdcace9aa)
![image](https://github.com/tgstation/tgstation/assets/44913068/bfa2a65d-49c8-40a9-b6dd-d4602494bada)
![image](https://github.com/tgstation/tgstation/assets/44913068/9ef2bcdd-aae1-4422-87b4-2c238399a9f0)
![image](https://github.com/tgstation/tgstation/assets/44913068/c89ec4eb-6ea5-4e0d-bdd9-4951b23fa580)
:cl:
add: Added navigation beacons on KS13 bots now work there
add: Added an axe to KS13
add: Added a supermatter from an off course supply pod to KS13
qol: Made it so you don't get stuck in the hole in engineering at KS13 as drone
fix: Fixed a duplicate window spawner at KS13
/:cl:

HourlyLimit committed:
Fixed, and Added small improvements to KS13

I made it so bots now work on station, there is now an axe to remove unwanted windows, i made it so you don't get stuck in the hole in engineering, and added a supermatter from a off course pod for faster station rebuilding so the round is not spent only setting up power.

HourlyLimit committed:
fix overlap

fix overlap of two lattices

AtomTheProphet commented:
YES, KS13 SUPERMATTER. I love you

LemonInTheDark commented:
why is there a supermatter in medbay did it crashland?

PeriodicChaos commented:
> why is there a supermatter in medbay did it crashland?

Yeah, I mean it can be anywhere (I'm going to fix that now with Var editing, pods are not letting me use them open by default)

HourlyLimit committed:
Add the pod

I had to reskin a closet because it turns out you cannot open a pod with a var. Nice

HourlyLimit committed:
Fix lint being mad

just moved it if anyone can tell me a better way id like that thanks.

LemonInTheDark commented on _maps/RandomRuins/SpaceRuins/russian_derelict.dmm:
```diff
@@ -7777,8 +7777,27 @@
+	base_icon_state = "pod";
+	can_weld_shut = 0;
+	opened = 1
+	},
```
these should be subtypes

HourlyLimit committed:
oops
